src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -771,6 +771,36 @@ def uploadRSEToMongoDB(self, rse, fullRSEToDB=False, overwrite=True):
             raise MSUnmergedPlineExit(msg)
         return rse
 
+    def getStatsFromMongoDB(self, detail=False, **kwargs):
+        """
+        Auxiliary method to serve the APIs of the REST calls for the service.
+        Implements various queries to MongoDB based on the parameters passed.
+        :param detail:           Bool marking if additional details must be queried from the database
+                                 (e.g. putting the 'dirs' field into the mongo projection)
+        :param rse=rseName:      String representing the RSE name to query the database for
```

Comment:
I dislike making another comment on this docstring, but it's still not consistent with the source code. Please make your best to provide correct docstrings for code freshly implemented.

Reply:
I do not get this comment.

Reply:
Now that rse_type is no longer used in the code, the docstring is consistency. Thanks